src/eslint-config-adeira/CHANGELOG.md

```diff
@@ -1,9 +1,9 @@
 # Unreleased
 
-- Rule `jest/prefer-expect-resolves` set to next version error
-- Rule `jest/prefer-to-be` set to next version error
```

Comment:
I changed the wording a little bit because "next version error" is an internal concept and unknown in the outside Eslint world.